Evergreen/Apps/Get-WinFSP.ps1

```diff
@@ -0,0 +1,28 @@
+Function Get-WinFSP {
+    <#
+        .SYNOPSIS
+            Get the current version and download URL for WinSFP.
+
+        .NOTES
+            Site: https://winfsp.dev/rel/
+            Author: Bill Zissimopoulos
```

Comment:
@alex-harvey-z3q optional, but this is typically your details as the contributor to the project for this app